playground/ssr-html/test-stacktrace.js

```diff
@@ -0,0 +1,47 @@
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { createServer } from 'vite'
+
+const isSourceMapEnabled = process.argv[2] === 'true'
+process.setSourceMapsEnabled(isSourceMapEnabled)
+console.log('# sourcemaps enabled:', isSourceMapEnabled)
```

Comment:
This won't work if something called `process.setSourceMapsEnabled` after this line. (I mean the `isSourceMapEnabled` will not be synced with the actual value.)

I've created a feature request to Node to fix this: https://github.com/nodejs/node/issues/46304